software/aleph.yml

```
@@ -0,0 +1,12 @@
+name: Aleph
+website_url: https://aleph.occrp.org/
+description: "Track people and companies."
```

The current description doesn't really explain what this project is about. (Open for other descriptions as well)
```suggestion
description: "Tool for indexing large amounts of both documents (PDF, Word, HTML) and structured (CSV, XLS, SQL) data for easy browsing and search."
```